lib/ansible/modules/windows/win_toast.ps1

```diff
@@ -0,0 +1,101 @@
+#!powershell
+# This file is part of Ansible
+#
+# Copyright 2017, Jon Hawkesworth (@jhawkesworth) <figs@unity.demon.co.uk>
+#
+# Ansible is free software: you can redistribute it and/or modify
```

Comment:
We now have a one-line GPL header for files.

Reply:
Thanks will fix that